app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -134,9 +133,7 @@ class SessionControlView(
     override val containerView: View?
```

Comment:
So we can't merge them both since `LayoutContainer` is an interface and the `containerView` comes from LayoutContainer ( which I haven't personally used but it seems to be a kotlin extension) and we need to override it. I'm open to ideas though 😄

Reply:
To be explicit, using the initializer I wrote above and removing the separate `containerView` declaration seemed to work locally.